README.md

```diff
@@ -15,8 +15,14 @@ public static IHostBuilder CreateHostBuilder(string[] args) =>
     .ConfigureWebHostDefaults(webBuilder => webBuilder.UseStartup<Startup>());
               
 ```
+Or from ASPNET 6 and onwards with minimal `Program.cs` setup:
 
-sThe `CompositionRoot` class is usually placed in the host project (the project containing `program.cs`) and is used to register services that are specific to **LightInject**
+```c#
+var builder = WebApplication.CreateBuilder(args);
+builder.Host.UseLightInject(sr => sr.RegisterFrom<CompositionRoot>());
+```
+
+The `CompositionRoot` class is usually placed in the host project (the project containing `program.cs`) and is used to register services that are specific to **LightInject**
```

Comment:
Uppercare `Program.cs` here an we are good 👍😃

Reply:
👍 👍